charts/cloudprem/README.md

```diff
@@ -636,6 +636,7 @@ Dex:
 | portal.config.cookie.secretKeys | object | `{"secret":""}` | Cookie secret key |
 | portal.config.environment | string | `"production"` | Portal environment |
 | portal.config.featuresDisabled[0] | string | `"console_v3_beta"` |  |
+| portal.config.stackUrl | string | `"https://#{organizationId}-#{stackId}.#{regionBaseUrl}/api"` | Stack URL |
```

Comment:
Why do we need that

Reply:
I've already seen the same things in console. Can't we add something else to handle this case

Reply:
The value may vary depending on the environment. Because it depends on where Stargate is deployed.